### PR TITLE
[ttLib] Add aliases for renamed entries in hhea table

### LINKED ISSUE #1714: ascender and ascent
The [opentype spec ](https://docs.microsoft.com/en-gb/typography/opentype/spec/hhea) calls the first two substantive entries in the `hhea` table "`ascender`" and "`descender`". fonttools calls them "`ascent`" and "`descent`".

This was surprising! Maybe it's too late to change then but can we at least have an alias?

### PR DESCRIPTION
**Pull request by simoncozens**
Closes #1714

### CODE REVIEW

**Comment by justvanrossum** [comment]
I wonder if it should be the other way around: rename the actual attributes and provide getters/setters for the old names?

**Comment by simoncozens** [comment]
Huh. Maybe not, as it would mean the XML files become incompatible.

**Comment by behdad** [comment]
XML as well as properties dict...

**Comment by justvanrossum** [comment]
> Maybe not, as it would mean the XML files become incompatible

Right, your original proposal is safest.

File: Tests/ttLib/tables/_h_h_e_a_test.py [hunk 1]
```diff
@@ -126,6 +126,13 @@ def test_toXML_version_as_float(self):
             len([r for r in captor.records
                  if "Table version value is a float" in r.msg]) == 1)
 
+    def test_aliases(self):
+        hhea = self.font['hhea']
+        self.assertEqual(hhea.ascent, hhea.ascender)
+        self.assertEqual(hhea.descent, hhea.descender)
+        hhea.ascender = 800
+        self.assertEqual(hhea.ascent, 800)
+        hhea.ascent = 750
```
Comment (justvanrossum) [inline Tests/ttLib/tables/_h_h_e_a_test.py]: Did you mean to continue here? You're not testing the descender setter.

**Comment by anthrotype** [comment]
thanks. I'll complete the test cases for the descender setter

**Pull request merged**